MaybeGame/EnemyStats.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+
+namespace MaybeGame
+{
+    public class EnemyStats
+    {
+        public int Strength { get; set; }
+        public int Stamina { get; set; }
+        public int Agility { get; set; }
+        public int Dexterity { get; set; }
+        public int Health { get; set; }
+        public int Defence { get; set; }
+        public double DodgeChance { get; set; }
+        public int Speed { get; set; }
+        public int AttackPower { get; set; }
+
+        public int SetStrength(string enemy)
+        {
+            // Strength is the quality or state of being physically strong.
+
+            var strength = 5;
+
+            switch (enemy)
+            {
+                case "Zombie":
+                    strength += 2;
+                    break;
+
+                case "Skeleton":
+                    strength += 2;
+                    break;
+
+                case "Bandit":
+                    strength += 3;
+                    break;
+
+                case "Theif":
+                    strength += 2;
+                    break;
+
+                case "Warrior":
+                    strength += 4;
+                    break;
+                
+                case "Berserker":
+                    strength += 7;
+                    break;
+                
+                case "Rat":
+                    strength += 0;
+                    break;
+
+                default:
+                    Console.WriteLine("Error 404: Race not found");
```

Comment:
404 is api response type, not applicable to console

Reply:
This is just a placeholder to let me know if there was some kind of error